website/docs/r/lb.html.markdown

```diff
@@ -135,6 +135,7 @@ Subnet Mapping (`subnet_mapping`) blocks support the following:
 * `subnet_id` - (Required) The id of the subnet of which to attach to the load balancer. You can specify only one subnet per Availability Zone.
 * `allocation_id` - (Optional) The allocation ID of the Elastic IP address.
 * `private_ipv4_address` - (Optional) A private ipv4 address within the subnet to assign to the internal-facing load balancer.
+* `ipv6_address` - (Optional) An ipv6 address within the subnet to assign to the internal-facing load balancer.
```

Comment:
the AWS api docs were a bit vague in the "SubnetMapping" definition but in the "CreateLoadbalancer" "SubnetMappings.member.N" reference section they seem to point out this arg is for internet facing NLBs (https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_CreateLoadBalancer.html) so just a minor update here 😃 
```suggestion
* `ipv6_address` - (Optional) An ipv6 address within the subnet to assign to the internet-facing load balancer.
```